packages/example-getting-started/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@loopback/example-getting-started",
+  "version": "1.0.0",
+  "description": "An application and tutorial on how to build with LoopBack 4.",
+  "main": "index.js",
+  "engines": {
+    "node": ">=8"
+  },
+  "scripts": {
+    "acceptance": "lb-dist mocha --opts ../../test/mocha.opts 'DIST/test/acceptance/**/*.js'",
```

Comment:
FWIW, `--opts ../../test/mocha.opts` will not work after this example project is cloned via `lb4 example` command. We need to fix `lb4 example` to extract monorepo-level `mocha.opts` into `test/mocha.opts` and fix all package.json scripts referring to `mocha.opts`. Since `test/mocha.opts` is the default location loaded by Mocha, we should simply remove `--opts` argument completely.

Feel free to leave this fix out of scope of this PR, I think this may be easier to test once the `master` branch contains an example project using `mocha.opts`.

Reply:
On the second thought, since we are already keeping almost everything build/test related in `@loopback/build`, it will be easier to move `mocha.opts` to that package too. See https://github.com/strongloop/loopback-next/pull/876.

If/when #876 is landed, then this line needs to be changed to `--opts node_modules/@loopback/build/mocha.opts`